Uppsala/UppEt20.xml

```diff
@@ -0,0 +1,524 @@
+                                        Den kallas Kedasie (det betyder heliggörelse)<lb/>
+                                        Den innehåller förutom ordning för olika<lb/>
+                                        gudstjänster en del böner och däribland<lb/>
+                                        fjorton litanior.
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <desc type="StampExlibris">Paper sticker on the inside of the front board.</desc>
+                                    <q xml:lang="sv">
+                                        Uppsala Universitets<lb/>
+                                        Bibliotek<lb/>
+                                        Handskr. avd.<lb/>
+                                        O Etiop. 20<lb/>
+                                    </q>
+                                </item>
+
+                            </list>
+                        </additions>
+                        
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1" type="Boards">Wooden boards with blind-tooled leather cover.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                        
+                    </physDesc>
+                                        
+
+                    <history>
+                        <origin>
+                            <origDate notBefore="1682" notAfter="1694">1682–1694</origDate>
+                        </origin>
+                        <provenance>
+                            The manuscript originally belonged to <persName ref="PRS14566ZawaldaMaryam"/> (see, e.g., <locus target="154ra 190v"/>).
+                            It was acquired by an unknown missionary of
+                            Evangeliska fosterlandsstiftelsen and brought to Sweden at some point between <date when="1866">1866</date>
+                            (when the first missionaries connected to Evangeliska fosterlandsstiftelsen arrived in Ethiopia) and
+                            <date when="1928">1928</date>, when it was catalogued in Sweden. During a period, 
+                            the manuscript was kept at <placeName ref="INS1020EFSmissionsmuseum"/> in 
+                            Johannelund (Ulvsunda,  <placeName ref="LOC7456Stockholm"/>). 
+                                After being catalogued by <persName ref="PRS6337Lofgren"/> in <date when="1928">1928</date>,
+                            apparently in Uppsala (<bibl><ptr target="bm:Loefgren1929Evangeliska"/><citedRange unit="page">1</citedRange>
+                            </bibl>),
+                            the manuscript was given by Evangeliska fosterlandsstiftelsen to <persName ref="PRS14565ErlingEidem">
```

Comment:
role="owner"


we assign roles to persons relevant for the manuscript history (donor, commissioner, scribe, owner)